PeripheryDemo/ViewController.swift

```diff
@@ -1,13 +1,20 @@
 //
 //  ViewController.swift
-//  Periphery Demo
+//  PeripheryDemo
 //
 //  Created by Sheikh Atif on 14/07/24.
 //
 
 import UIKit
 
+class Useless{
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/SheikhAtifIBM/Periphery-Demo/blob/8ae70dbe002b70726bb9e9b9dbf964a049fbb194/PeripheryDemo/ViewController.swift#L10"></span>Class 'Useless' is unused</td>
    </tr>
  </tbody>
</table>